src/jdk.jpackage/share/man/jpackage.md

```diff
@@ -423,7 +423,7 @@ The `jpackage` tool will take as input a Java application and a Java run-time im
 
 :   Menu group this application is placed in
 
-<a id="option-linux-package-deps">`--linux-package-deps`</a>
+<a id="option-linux-package-deps">`--linux-package-deps *package-dep* \[`,`*package-dep*...\]`</a>
```

Comment:
Should be \*package-dep-string\* to sync with HelpResources.properties

Reply:
```suggestion
<a id="option-linux-package-deps">`--linux-package-deps *package-dep-string*`</a>
```